BedrockServer.cpp

```diff
@@ -2142,16 +2148,16 @@ void BedrockServer::_acceptSockets() {
     }
 }
 
-void BedrockServer::waitForHTTPS(BedrockCommand&& command) {
+void BedrockServer::waitForHTTPS(unique_ptr<BedrockCommand>&& command) {
     lock_guard<mutex> lock(_httpsCommandMutex);
 
-    // Create a new BedrockCommand on the head via moving from our existing command. This is the one we'll store.
-    BedrockCommand* commandPtr = new BedrockCommand(move(command));
+    // Un-uniquify the unique_ptr. I don't love this, but it works better with the code we've already got.
```

Comment:
what's the alternative here?

Reply:
The alternative is to combine `_outstandingHTTPSCommands` and `_outstandingHTTPSRequests` into one data structure that stores only a single copy of the command pointer, but is searchable by both command pointer and transaction. Which is doable but obviously more work.

Reply:
can we do that as a follow up? :D